src/sortStudents.ts

```diff
@@ -1,16 +1,65 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function calcAverage(grades: number[]): number {
+  const sumOfGrades: number = grades.reduce((sum: number, current: number) => (
```

Comment:
```suggestion
  return grades.reduce((sum, element) => sum + element, 0) / grades.length;
```
you could simplify the function, into a stright forward return with the value that you need.